static/css/sass/cdr_ui_styles.scss

```diff
@@ -32,6 +33,18 @@ $box-shadow: inset 3px 3px 10px -1px $box-shadow-color, 0px 16px 22px -15px $box
   }
 }
 
+/* Class to display content for screen readers */
+.sr-only {
```

Comment:
is this all standard for `sr-only` elements?

Reply:
I stole it from bootstrap. It's what they use.